flang/lib/Optimizer/Transforms/SimplifyRegionLite.cpp

```diff
@@ -0,0 +1,46 @@
+//===- SimplifyRegionLite.cpp -- region simplification lite ---------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "PassDetail.h"
+#include "flang/Optimizer/Dialect/FIROps.h"
+#include "flang/Optimizer/Transforms/Passes.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "mlir/Transforms/RegionUtils.h"
+
+namespace {
+
+class SimplifyRegionLitePass
+    : public fir::SimplifyRegionLiteBase<SimplifyRegionLitePass> {
+public:
+  void runOnOperation() override;
+};
+
+class DummyRewriter : public PatternRewriter {
+  public:
+    DummyRewriter(MLIRContext *ctx) : PatternRewriter(ctx) {}
```

Comment:
My pretty printer has these lines two spaces to the left.

Reply:
Nice catch, Mr. Eagle-eyes.